packages/core/src/base/icons.scss

```diff
@@ -0,0 +1,207 @@
+/* stylelint-disable selector-class-pattern */
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Icons
+
+Iconography should be used sparingly and only with support of labels. We use a subset of FontAwesome icons, with one or two bespoke icons.
+
+Please refer to [USWDS' documentation](https://designsystem.digital.gov/components/icons/) for more detail.
+
+Style guide: style.icons
+*/
+
+/*
+Alert Icons
+
+@hide-markup
+
+These icons are used primarily as part of the Alert Box component, though the Checkmark can sometimes be found in other places, without the background.
+
+Markup:
+<% var icons = [
+  'information', 'success', 'success-fill', 'warning', 'error'
+]; -%>
+<div class="ds-l-container ds-u-padding--1 ">
```

Comment:
We'll need to remove the `ds-l-container` class from these divs to get the icons pulled to the left like in the screenshots provided